developer_setup.md

```diff
@@ -277,6 +288,14 @@ targeted end-user testing. See the [minimal mode guide](developer_setup/minimal_
 bin/update                # Updates dependencies using bundler and bower, runs migrations, prepares test db.
 ```
 
+### Upgrading tools after an OS upgrade
+
+* Reinstall xcode developer tools
+* Uninstall existing ruby version with your version manager
```

Comment:
Why?

Reply:
Because Jason said so!

Reply:
Why not? Why to which part?

Reply:
Why would you delete and reinstall your ruby?